libasn1print/asn1printproto.c

```diff
@@ -1640,6 +1634,38 @@ proto_process_children(asn1p_expr_t *expr, proto_msg_t *msgdef, int repeated, in
 								se->expr_type, se->meta_type, expr->Identifier, se->Identifier);
 					}
 				}
+				if (oneof) {
```

Comment:
why do we need this piece of code ?

Reply:
This is to fix representation of SEQUENCE OF in Protobuf (it also requires nested messages).
This little part of code is not related to memory leak issue

Reply:
ok